src/Page.js

```diff
@@ -129,7 +129,7 @@ function formatFooter(pageData) {
   return $.html();
 }
 
-function formatSiteNav(renderedSiteNav) {
+function formatSiteNav(renderedSiteNav, src) {
```

Comment:
> The first time we render the siteNav for a page, we pass in `this.src` so that `formatSiteNav` can locate the link to set `class=current` to at `Page.js#143`. When we recursively format the same siteNav for nested lists in `Page.js#157` and `Page.js#184`, we do not pass in the `src` parameter since we do not need to highlight the same link repeatedly for the same page.

I see. However, we don't seem to use the content of `src` at all in the method, I wonder if we can substitute it with a boolean value instead?

Reply:
No, because we actually do use the value of `src` at `Page.js#143` to find the correct link to highlight:

```js
listItems.find(`a[href='{{baseUrl}}/${src.replace(/\.(md|mbd)$/, '.html')}']`)
      .attr('class', 'current');
```